docs/src/content/docs/tr/guides/authoring-content.md

```diff
@@ -0,0 +1,209 @@
+
+```md
+---
+title: Sayfa İçeriğim
+description: Starlight'ın kurulu bağlantıları nasıl kullanılır
+---
+
+## Giriş
+
+[Görüşümü](#conclusion) aynı sayfanın aşağısına iliştirebilirim.
+
+## Görüş
+
+`https://my-site.com/page1/#introduction` giriş bölümüme direkt olarak yönlendirir.
+```
+
+Seviye 2 (`<h2>`) ve Seviye 3 (`<h3>`) başlıklar otomatik olarak içerik tablosunda görünecektir.
+
+## Ara Bölümler
+
+Ara bölümler, sayfanın ana içeriğinin yanında ikincil bilgi gösterimi için kullanışlıdır.
+
+Starlight ara bölümleri oluşturmak için özel Markdown sözdizimi sunar. Ara bölüm blokları üç adet iki nokta üst üste'nin `:::` içeriği sarmalamasıyla kullanılır ve tip olarak `note`,`tip`, `caution` ya da `danger` kullanılabilir.
+
+Herhangi bir Markdown içerik tipini ara bölümü içerisine yerleştirebilirsiniz, ancak ara bölümler kısa ve öz içerikler için biçilmiş kaftandır.
+
+### Note Ara bölümü
+
+:::note
+Starlight, [Astro](https://astro.build/) ile oluşturulmuş bir dokümantason website oluşturma aracıdır. Bu komutla başlayabilirsiniz:
+
+```sh
+npm create astro@latest -- --template starlight
+```
+
+:::
+
+````md
+:::note
+Starlight, [Astro](https://astro.build/) ile oluşturulmuş bir dokümantason website oluşturma aracıdır. Bu komutla başlayabilirsiniz:
+
+```sh
+npm create astro@latest -- --template starlight
+```
+
+:::
+````
+
+### Özel ara bölümler
+
+Ara bölümler için ara bölüm tipinin tanımından hemen sonra, köşeli parantez arasında olacak şekilde ara bölümlerinizi özelleştirebilirsiniz (örn.`:::tip[Bunu biliyor musun?]`).
+
+:::tip[Bunu biliyor musun?]
+Astro [Ada Mimarisi”](https://docs.astro.build/en/concepts/islands/) ile daha hızlı websitesi oluşturmana yardımcı olur.
+:::
+
+```md
+:::tip[Bunu biliyor musun?]
+Astro [Ada Mimarisi”](https://docs.astro.build/en/concepts/islands/) ile daha hızlı websitesi oluşturmana yardımcı olur.
+:::
+```
+
+### Diğer Ara bölümler
+
+Uyarı ve tehlike ara bölümleri, kullanıcıların dikkatini gözden kaçabilecek detaylara çekmek için kullanışlıdır.
+Bunları çok kullandığınızı farkederseniz, dokümanınızın yeniden oluşturulmasına gerek kalmayacağının işareti olabilir.
+
+:::caution[Uyarı]
+Harika bir dokümantasyon sitesi istediğine emin değilsen, [Starlight](../../) kullanmadan önce iki kez düşün.
+:::
+
+:::danger[Tehlike]
+Yardımcı Starlight özellikleri sayesinde kullanıcılarınız daha kolay ürün bulabilir ve daha üretken olabilir.
+
+- Yönlendirmeyi temizle
+- Kullanıcı-yapılandırmalı renk teması
+- [i18n desteği](/guides/i18n)
+
+:::
+
+```md
+:::caution
+Harika bir dokümantasyon sitesi istediğine emin değilsen, [Starlight](../../) kullanmadan önce iki kez düşün.
+:::
+
+:::danger
+Yardımcı Starlight özellikleri sayesinde kullanıcılarınız daha kolay ürün bulabilir ve daha üretken olabilir.
+
+- Yönlendirmeyi temizle
+- Kullanıcı-yapılandırmalı renk teması
+- [i18n desteği](/guides/i18n)
```

Comment:
These links should also be to the `/tr/` page:

````suggestion
- [i18n desteği](/tr/guides/i18n)

:::

```md
:::caution
Harika bir dokümantasyon sitesi istediğine emin değilsen, [Starlight](../../) kullanmadan önce iki kez düşün.
:::

:::danger
Yardımcı Starlight özellikleri sayesinde kullanıcılarınız daha kolay ürün bulabilir ve daha üretken olabilir.

- Yönlendirmeyi temizle
- Kullanıcı-yapılandırmalı renk teması
- [i18n desteği](/tr/guides/i18n)
````